main.go

```diff
@@ -1,17 +1,14 @@
-package main
+package ironio
```

Comment:
The root package must be called `main` to avoid errors once Terraform initializes the provider.